src/main/scala/com/ambiata/disorder/GenPlus.scala

```diff
@@ -5,8 +5,11 @@ import scala.collection.JavaConverters._
 
 object GenPlus {
 
-  def choose(from: Int, to: Int): Gen[Int] =
-    Gen.sized(n => Gen.choose(from, Math.max(from, Math.min(n, to))))
+  def chooseInt(from: Int, to: Int): Gen[Int] =
```

Comment:
If this was being used externally more I might avoid renaming it, but I think you'll be fine